app/src/main/AndroidManifest.xml

```diff
@@ -16,8 +16,43 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".Main3Activity" />
-        <activity android:name=".Main4Activity" />
+        <activity
+            android:name=".Activity_of_Learning_Activity"
+            android:label="Activity_of_Learning_Activity" />
+        <activity
+            android:name=".Activity_of_Learning_Activity_Lifecycle_1"
+            android:label="Activity_of_Learning_Activity_Lifecycle_1" />
+        <activity android:name=".Activity_of_Learning_Activity_Lifecycle_2" />
+        <activity
+            android:name=".Activity_of_Learning_Tasks"
```

Comment:
Same as mentioned above. Use meaningful names without underscore. Use sentence case for eg: LearningLifecycle